samples/unit-tests/indicator-price-indicator/recalculations/demo.js

```diff
@@ -318,4 +318,84 @@ QUnit.test('The lastPrice color, #15074.', function (assert) {
         '#00ff00',
         'The lastPrice color should remain the same after toggle.'
     );
+});
+
+QUnit.test('The currentPriceIndicator for multiple series, #14888.', function (assert) {
+    const chart = Highcharts.stockChart('container', {
+            yAxis: [{
+                height: '60%'
+            }, {
+                top: '65%',
+                height: '35%'
+            }],
+            stockTools: {
+                gui: {
+                    enabled: true,
+                    buttons: [
+                        "currentPriceIndicator"
+                    ]
+                }
+            },
+            series: [{
+                id: 'main',
+                color: '#00ffff',
+                data: [2, 6, 8, 6, 3, 1, 1, 3, 5, 6, 9, 9, 9, 7, 4, 2, 1, 9]
+            }, {
+                type: 'sma',
+                linkedTo: 'main',
+                color: '#ff00ff'
+            }, {
+                type: 'column',
+                color: '#000000',
+                yAxis: 1,
+                data: [10, 2, 5, 6, 1, 3, 5, 1, 3, 5, 4, 1, 3, 5, 6, 4, 1, 4]
+            }]
+        }),
+        button = chart.stockTools.listWrapper.childNodes[0].childNodes[0];
+
+    // Click the currentPriceIndicator button in the stock tools.
+    chart.navigationBindings.options.bindings.currentPriceIndicator.init.call(
+        chart.navigationBindings, button);
+
+    assert.strictEqual(
+        chart.series[0].lastPrice.attr('stroke'),
+        chart.series[0].color,
+        'The first series lastPrice line should have color as series.'
+    );
+    assert.strictEqual(
+        chart.series[0].crossLabel.attr('fill'),
+        chart.series[0].color,
+        'The first series lastVisiblePrice label should have color as series.'
+    );
+    assert.strictEqual(
+        chart.series[1].lastPrice.attr('stroke'),
+        chart.series[1].color,
+        'Indicator lastPrice line should have color as series.'
+    );
+    assert.strictEqual(
+        chart.series[1].crossLabel.attr('fill'),
+        chart.series[1].color,
+        'Indicator lastVisiblePrice label should have color as series.'
+    );
+    assert.strictEqual(
+        chart.series[1].lastPrice.attr('stroke'),
+        chart.series[1].color,
+        'Volume lastPrice line should have color as series.'
+    );
+    assert.strictEqual(
+        chart.series[1].crossLabel.attr('fill'),
+        chart.series[1].color,
+        'Volume lastVisiblePrice label should have color as series.'
+    );
```

Comment:
You test twice the same series, probably should be `series[2]`.

How about placing these tests in `forEach()`?